.circleci/config.yml

```diff
@@ -119,6 +119,18 @@ jobs:
       - run: go run build/ci.go test -coverage
       - run: bash <(curl -s https://codecov.io/bash)
 
+  coverage_istanbul:
```

Comment:
Can we mention `e2e` here? e.g. `coverage_istanbul_for_e2e`

Reply:
I think I'll remove it from circleci since I don't think we can have both reports in the PR, so I'll just create a `make` target for it

Reply:
right